src/families/stellar/memo-type-check.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import { makeLRUCache } from "../../cache";
+import axios from "axios";
+
+const baseUrl: string = "https://api.stellar.expert/";
+
+//$FlowFixMe <--- Don't know how to resolve this properly
+const memoTypeCheckCache = makeLRUCache(
```

Comment:
What's the error ?

Reply:
> Missing type annotation for `A`. `A` is a type parameter declared in  function [1] and was implicitly instantiated at  call of `makeLRUCache` [2].Flow(InferError)

Reply:
Fixed (flow doesn't seem to infer correctly)